docs/overview/why-anvil.mdx

```diff
@@ -0,0 +1,275 @@
+---
+description: "A guide answering the question of why researchers and educators should use the AnVIL"
+title: "Why use AnVIL?"
+---
+
+# Why use AnVIL?
+
+The NHGRI AnVIL (Genomic Data Science Analysis, Visualization, and Informatics Lab-space) is a project powered by Terra for biomedical researchers to access data, run analysis tools, and collaborate. Both biology researchers and educators can benefit from using AnVIL ([anvil.terra.bio](https://anvil.terra.bio/)) for their research and in the classroom.
+
+This guide acts as a resource answering the question "why use AnVIL?". It will discuss the research, classroom, and general benefits of using AnVIL and point to related resources throughout.
+
+{/*
+  Image outlining this guide
+*/}
```

Comment:
For my own knowledge, are these comment blocks TODOs to be completed at a later date?

Reply:
Got it, thanks Kate! (My apologies, I completely missed your questions above.) I'll resolve this conversation and add my answers below.